deployments/nasa/hubploy.yaml

```diff
@@ -8,9 +8,10 @@ images:
 
 cluster:
   provider: aws
+    project: pangeo-nasa-access
+    service_key: aws-config.txt
   gcloud:
     project: TBD
     service_key: TBD
     cluster: pangeo
     zone: us-east-1
```

Comment:
should we remove the gcloud stuff here?

Reply:
Yeah, it's unnecessary. I can pull it out.